exec/java-exec/src/main/java/org/apache/drill/exec/util/Utilities.java

```diff
@@ -43,19 +43,21 @@ public static String getFileNameForQueryFragment(FragmentContext context, String
   }
 
   /**
-   * Create QueryContextInformation with given <i>defaultSchemaName</i>. Rest of the members of the
+   * Create QueryContextInformation with given <i>defaultSchemaName</i> and <i>sessionId</i>. Rest of the members of the
    * QueryContextInformation is derived from the current state of the process.
    *
    * @param defaultSchemaName
+   * @param sessionId
    * @return
    */
```

Comment:
Please add description for two @param and return, since you are changing method description.